starwars/src/components/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
ReactDOM isn't necessary in this file. It's only used in index.js to connect our React app to the HTML -> DOM